### DISCUSSION TITLE
Can you add  Sticky Side 3.3.4 to cdnjs

### DISCUSSION THREAD

**Discussion by BeRGB** [cdnjs/cdnjs#14234] [General]
Hi guys,

can you add latest version from git, 3.3.1  has bugs :)

**Comment by MattIPv4** [comment]
The library is configured to pull updates from NPM: https://github.com/cdnjs/packages/blob/master/packages/s/sticky-sidebar.json

When a new version is published to NPM, we will automatically add it to the CDN.
